tests/functional/rbac/test_reassignment.py

```diff
@@ -149,6 +154,137 @@ def grant_role(self, client: ADCMClient, user: User, role: RbacRoles, *objects:
                 name=f'{user.username} is {role.value}', role=client.role(name=role.value), objects=objects, user=[user]
             )
 
+    def test_add_remove_user_from_group_and_policy(self, clients, is_denied_to_user, prepare_objects, user):
+        """Test that policies are applied after user add/remove to the policy"""
```

Comment:
A little bit of inconsistency:
the name tells us that the user is added/removed from group and policy
the docstring mentions only policy
in the test body, there's no adding user to the group after policy assignment.

I suggest minor docstring change, like "user add/remove to the policy **directly or as a part of a group**"